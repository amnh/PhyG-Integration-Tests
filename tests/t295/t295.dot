digraph G {
	rankdir = LR;	edge [colorscheme=spectral11];	node [shape = none];

    0 [label=AlbanianTosk];
    1 [label=AncientIonicGreek];
    2 [label=ArmenianWestern];
    3 [label=Assamese];
    4 [label=Cornish];
    5 [label=Danish];
    6 [label=EasternFrisianSaterland];
    7 [label=Faroese];
    8 [label=Icelandic];
    9 [label=Irish];
    10 [label=KolschColognian];
    11 [label=Manx];
    12 [label=OldPrussian];
    13 [label=SanskritVedic];
    14 [label=Scots];
    15 [label=Yiddish];
    16 [label=HTU16];
    17 [label=HTU17];
    18 [label=HTU18];
    19 [label=HTU19];
    20 [label=HTU20];
    21 [label=HTU21];
    22 [label=HTU22];
    23 [label=HTU23];
    24 [label=HTU24];
    25 [label=HTU25];
    26 [label=HTU26];
    27 [label=HTU27];
    28 [label=HTU28];
    29 [label=HTU29];
    30 [label=HTU30];
    16 -> 0 [label=0.0];
    16 -> 23 [label=0.0];
    17 -> 2 [label=1167.2];
    17 -> 19 [label=1278.0];
    18 -> 29 [label=86.8];
    18 -> 30 [label=233.5];
    19 -> 13 [label=2621.0];
    19 -> 15 [label=2409.2];
    20 -> 8 [label=5692.0];
    20 -> 11 [label=1046.9];
    21 -> 1 [label=4708.1];
    21 -> 4 [label=857.2];
    22 -> 5 [label=589.4];
    22 -> 18 [label=86.1];
    23 -> 6 [label=702.9];
    23 -> 25 [label=241.0];
    24 -> 10 [label=5985.0];
    24 -> 22 [label=316.1];
    25 -> 14 [label=855.4];
    25 -> 24 [label=614.3];
    26 -> 17 [label=212.0];
    26 -> 20 [label=370.0];
    27 -> 7 [label=817.0];
    27 -> 9 [label=970.6];
    28 -> 3 [label=3944.4];
    28 -> 27 [label=399.1];
    29 -> 12 [label=831.7];
    29 -> 21 [label=211.0];
    30 -> 26 [label=47.0];
    30 -> 28 [label=197.3];
}
//48163.2
digraph G {
	rankdir = LR;	edge [colorscheme=spectral11];	node [shape = none];

    0 [label=AlbanianTosk];
    1 [label=AncientIonicGreek];
    2 [label=ArmenianWestern];
    3 [label=Assamese];
    4 [label=Cornish];
    5 [label=Danish];
    6 [label=EasternFrisianSaterland];
    7 [label=Faroese];
    8 [label=Icelandic];
    9 [label=Irish];
    10 [label=KolschColognian];
    11 [label=Manx];
    12 [label=OldPrussian];
    13 [label=SanskritVedic];
    14 [label=Scots];
    15 [label=Yiddish];
    16 [label=HTU16];
    17 [label=HTU17];
    18 [label=HTU18];
    19 [label=HTU19];
    20 [label=HTU20];
    21 [label=HTU21];
    22 [label=HTU22];
    23 [label=HTU23];
    24 [label=HTU24];
    25 [label=HTU25];
    26 [label=HTU26];
    27 [label=HTU27];
    28 [label=HTU28];
    29 [label=HTU29];
    30 [label=HTU30];
    16 -> 0 [label=0.0];
    16 -> 23 [label=0.0];
    17 -> 2 [label=1167.2];
    17 -> 19 [label=1278.0];
    18 -> 29 [label=86.8];
    18 -> 30 [label=233.5];
    19 -> 13 [label=2621.0];
    19 -> 15 [label=2409.2];
    20 -> 8 [label=5692.0];
    20 -> 11 [label=1046.9];
    21 -> 1 [label=4708.1];
    21 -> 4 [label=857.2];
    22 -> 5 [label=589.4];
    22 -> 18 [label=86.1];
    23 -> 6 [label=702.9];
    23 -> 25 [label=241.0];
    24 -> 10 [label=5985.0];
    24 -> 22 [label=316.1];
    25 -> 14 [label=855.4];
    25 -> 24 [label=614.3];
    26 -> 17 [label=212.0];
    26 -> 20 [label=370.0];
    27 -> 7 [label=817.0];
    27 -> 9 [label=970.6];
    28 -> 3 [label=3944.4];
    28 -> 27 [label=399.1];
    29 -> 12 [label=831.7];
    29 -> 21 [label=211.0];
    30 -> 26 [label=47.0];
    30 -> 28 [label=197.3];
}
//48163.2
