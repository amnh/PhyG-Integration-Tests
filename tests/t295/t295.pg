--build(graph), inputting all character types
set(seed:1634561640)
set(graphtype:hardwired)
read(rename: "woman_rename.txt")
read(aminoacid:"woman-wide_aa.fas", tcm:"dayhoff.mat")
read(fasta:"woman_12Saln.fasta", tcm:(1,2))
read(fasta:"woman_binary.fasta", tcm:(2,1))
read(prefasta:"woman_cox1aln.fasta", tcm:"sg2t1.mat")
read(fastc:"woman.fastc", tcm: "woman-11.tcm")
read(prefastc:"woman_cox1_spaced.fastc")
read(include:"Include_woman16.txt")
--woman_allchars.ss includes cc+, cc-, continuous, ?, [ ], /N 
read("woman_allchars.ss")
read("woman_step.ss")
build(character, replicates:10, block, displaytrees:5, atrandom, graph)
report("t295.csv", data, overwrite)
report("t295.tre", newick, overwrite)
report("t295.dot", dotpdf, overwrite)
