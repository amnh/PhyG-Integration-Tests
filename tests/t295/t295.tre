(AlbanianTosk:1292.85,(AncientIonicGreek:6672.6,(Danish:4419.95,((Scots:1666.0,(EasternFrisianSaterland:31030.0)#HTU30:31030.0)HTU25:3109.6,(((Assamese:3554.05,Icelandic:5259.1)HTU26:2530.6,((KolschColognian:5716.1,Yiddish:2691.95)HTU24:2256.4,#HTU30:31485.0)HTU36:1555.1)HTU33:2608.6,(((Manx:1338.3000000000002,(Irish:32780.0)#HTU31:32780.0)HTU19:530.45,(Cornish:28805.0)#HTU28:28805.0)HTU18:1382.6,(ArmenianWestern:2796.45,(SanskritVedic:4457.55,(((Faroese:1888.05,OldPrussian:2050.95)HTU20:542.7,#HTU31:32730.0)HTU21:625.45,#HTU28:29002.0)HTU22:1003.6)HTU23:821.6)HTU32:1074.6)HTU34:2376.8)HTU35:1831.6)HTU17:1978.45)HTU29:2095.4)HTU27:4341.35)HTU16[51211.3];
(AlbanianTosk:1284.85,(AncientIonicGreek:6187.1,(Danish:3742.95,(Scots:3942.5,(((Assamese:3514.05,Icelandic:5312.1)HTU23:2486.1,(EasternFrisianSaterland:2440.95,(KolschColognian:5721.6,Yiddish:2701.45)HTU22:2207.9)HTU30:1539.1)HTU27:2294.6,(Manx:2293.95,(ArmenianWestern:2752.95,(SanskritVedic:4425.55,(Cornish:2115.6,(Irish:1612.6,(Faroese:1890.55,OldPrussian:2051.45)HTU18:542.7)HTU19:624.45)HTU20:1032.1)HTU21:848.6)HTU26:673.3000000000001)HTU28:2124.6)HTU29:1596.6)HTU17:1740.95)HTU25:1782.4)HTU24:3780.35)HTU16[47227.7];
(AlbanianTosk:1286.35,(AncientIonicGreek:6189.6,(Danish:3738.45,(Scots:3919.7999999999997,(((Assamese:3533.05,Icelandic:5307.1)HTU23:2484.1,(EasternFrisianSaterland:2371.45,(KolschColognian:5722.6,Yiddish:2696.45)HTU22:2272.9)HTU30:1546.6)HTU27:2279.3,((Cornish:1433.6,Manx:1166.65)HTU18:1311.1,(ArmenianWestern:2479.95,(SanskritVedic:4335.45,(Irish:1657.7,(Faroese:1778.55,OldPrussian:2215.45)HTU19:564.1)HTU20:655.45)HTU21:712.2)HTU26:590.1)HTU28:2035.6)HTU29:1554.6)HTU17:1746.1499999999999)HTU25:1771.4)HTU24:3782.85)HTU16[46562.899999999994];
(AlbanianTosk:1296.35,(AncientIonicGreek:6042.1,(Danish:3681.95,((EasternFrisianSaterland:1551.95,Scots:1610.5)HTU23:2430.4,(((KolschColognian:5342.6,Yiddish:2858.95)HTU22:1826.6,(Assamese:3973.75,Icelandic:4621.4)HTU24:1658.1)HTU28:3333.1,(Manx:2331.1499999999996,(ArmenianWestern:2770.45,(SanskritVedic:4452.05,(Cornish:2130.1,(Irish:1625.6,(Faroese:1920.55,OldPrussian:2067.45)HTU18:547.2)HTU19:624.95)HTU20:1038.6)HTU21:856.6)HTU27:634.6)HTU29:1814.1)HTU30:1504.1)HTU17:1635.1499999999999)HTU26:1753.9)HTU25:3627.85)HTU16[45791.399999999994];
(AlbanianTosk:1296.35,(AncientIonicGreek:6042.1,(Danish:3681.95,((EasternFrisianSaterland:1551.95,Scots:1610.5)HTU23:2430.4,(((KolschColognian:5342.6,Yiddish:2858.95)HTU22:1826.6,(Assamese:3973.75,Icelandic:4621.4)HTU24:1658.1)HTU28:3333.1,(Manx:2331.1499999999996,(ArmenianWestern:2770.45,(SanskritVedic:4452.05,(Cornish:2130.1,(Irish:1625.6,(Faroese:1920.55,OldPrussian:2067.45)HTU18:547.2)HTU19:624.95)HTU20:1038.6)HTU21:856.6)HTU27:634.6)HTU29:1814.1)HTU30:1504.1)HTU17:1635.1499999999999)HTU26:1753.9)HTU25:3627.85)HTU16[45791.399999999994];
(AlbanianTosk:1293.85,(AncientIonicGreek:6040.6,(Danish:3676.95,((EasternFrisianSaterland:1562.45,Scots:1601.0)HTU23:2440.1,(((KolschColognian:5340.1,Yiddish:2871.45)HTU22:1810.1,(Assamese:3995.25,Icelandic:4626.9)HTU24:1656.1)HTU28:3355.9,((Cornish:1443.1,Manx:1237.15)HTU18:1380.1,(ArmenianWestern:2504.95,(SanskritVedic:4355.45,(Irish:1670.7,(Faroese:1806.55,OldPrussian:2234.45)HTU19:577.1)HTU20:653.45)HTU21:703.7)HTU27:566.6)HTU29:1681.3000000000002)HTU30:1482.1)HTU17:1631.95)HTU26:1748.4)HTU25:3642.85)HTU16[45107.9];
