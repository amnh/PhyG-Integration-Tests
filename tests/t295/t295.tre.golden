(AlbanianTosk:1125.05,(ArmenianWestern:3957.55,(Danish:3803.1499999999996,((Irish:1580.1999999999998,(Manx:31313.0)#HTU34:31313.0)HTU30:2577.0,(((Yiddish:3508.6,(AncientIonicGreek:4553.5,(Faroese:22690.5)#HTU28:22690.5)HTU22:1166.1)HTU23:1275.25,#HTU34:31423.5)HTU31:2451.6,(((OldPrussian:2429.9,Scots:1457.1)HTU24:388.6,#HTU28:22413.0)HTU25:2008.5,((Icelandic:3016.1,KolschColognian:3689.4)HTU17:4464.1,(EasternFrisianSaterland:2638.95,(Assamese:2399.2,(Cornish:4574.55,SanskritVedic:1876.6)HTU19:1485.2)HTU20:2217.45)HTU21:1718.6)HTU29:1863.1)HTU32:1956.1)HTU33:1891.45)HTU18:2007.1)HTU27:2217.0)HTU26:3744.05)HTU16[48521.4];
(AlbanianTosk:1112.5,(ArmenianWestern:3989.65,(Danish:3653.25,(Irish:3469.5,((Manx:1746.75,(AncientIonicGreek:3903.1,Yiddish:3512.0)HTU22:1691.05)HTU28:2586.0,((Faroese:1136.1,(OldPrussian:2380.9,Scots:1497.6)HTU23:358.1)HTU24:2040.5,((Icelandic:3057.1,KolschColognian:3659.4)HTU17:4583.6,(EasternFrisianSaterland:2661.45,(Assamese:2402.2,(Cornish:4574.55,SanskritVedic:1879.6)HTU19:1487.7)HTU20:2178.95)HTU21:1642.1)HTU27:1817.1)HTU29:2029.4)HTU30:1948.45)HTU18:1836.1)HTU26:2246.8)HTU25:3748.5)HTU16[44720.7];
(AlbanianTosk:1112.5,(ArmenianWestern:3989.65,(Danish:3653.25,(Irish:3469.5,((Manx:1746.75,(AncientIonicGreek:3903.1,Yiddish:3512.0)HTU22:1691.05)HTU28:2586.0,((Faroese:1136.1,(OldPrussian:2380.9,Scots:1497.6)HTU23:358.1)HTU24:2040.5,((Icelandic:3057.1,KolschColognian:3659.4)HTU17:4583.6,(EasternFrisianSaterland:2661.45,(Assamese:2402.2,(Cornish:4574.55,SanskritVedic:1879.6)HTU19:1487.7)HTU20:2178.95)HTU21:1642.1)HTU27:1817.1)HTU29:2029.4)HTU30:1948.45)HTU18:1836.1)HTU26:2246.8)HTU25:3748.5)HTU16[44720.7];
(AlbanianTosk:1112.5,(ArmenianWestern:3989.65,(Danish:3653.25,(Irish:3469.5,((Manx:1746.75,(AncientIonicGreek:3903.1,Yiddish:3512.0)HTU22:1691.05)HTU28:2586.0,((Faroese:1136.1,(OldPrussian:2380.9,Scots:1497.6)HTU23:358.1)HTU24:2040.5,((Icelandic:3057.1,KolschColognian:3659.4)HTU17:4583.6,(EasternFrisianSaterland:2661.45,(Assamese:2402.2,(Cornish:4574.55,SanskritVedic:1879.6)HTU19:1487.7)HTU20:2178.95)HTU21:1642.1)HTU27:1817.1)HTU29:2029.4)HTU30:1948.45)HTU18:1836.1)HTU26:2246.8)HTU25:3748.5)HTU16[44720.7];
(AlbanianTosk:1117.55,(ArmenianWestern:3993.05,(Danish:3788.6499999999996,((Irish:1528.6999999999998,Manx:1490.1999999999998)HTU28:2646.0,((AncientIonicGreek:3802.7999999999997,Yiddish:3536.3)HTU22:3145.55,((Faroese:1154.6,(OldPrussian:2384.9,Scots:1553.6)HTU23:422.1)HTU24:2384.0,((Icelandic:3081.6,KolschColognian:3687.9)HTU17:4212.1,(EasternFrisianSaterland:2831.45,(Assamese:2402.7,(Cornish:4586.55,SanskritVedic:1882.1)HTU19:1481.2)HTU20:1953.45)HTU21:1396.1)HTU27:1918.6)HTU29:1973.1)HTU30:2450.95)HTU18:1988.6)HTU26:2209.0)HTU25:3765.05)HTU16[44540.1];
(AlbanianTosk:1117.55,(ArmenianWestern:3993.05,(Danish:3788.6499999999996,((Irish:1528.6999999999998,Manx:1490.1999999999998)HTU28:2646.0,((AncientIonicGreek:3802.7999999999997,Yiddish:3536.3)HTU22:3145.55,((Faroese:1154.6,(OldPrussian:2384.9,Scots:1553.6)HTU23:422.1)HTU24:2384.0,((Icelandic:3081.6,KolschColognian:3687.9)HTU17:4212.1,(EasternFrisianSaterland:2831.45,(Assamese:2402.7,(Cornish:4586.55,SanskritVedic:1882.1)HTU19:1481.2)HTU20:1953.45)HTU21:1396.1)HTU27:1918.6)HTU29:1973.1)HTU30:2450.95)HTU18:1988.6)HTU26:2209.0)HTU25:3765.05)HTU16[44540.1];
